code/game/objects/structures/mirror.dm

```diff
@@ -86,7 +86,7 @@
 	name = "magic mirror"
 	desc = "Turn and face the strange... face."
 	icon_state = "magic_mirror"
-	var/list/races_blacklist = list("skeleton", "agent", "angel", "military_synth")
+	var/list/races_blacklist = list("skeleton", "agent", "angel", "military_synth", "synth", "goofzombies", "memezombies")

```

Comment:
Sorry I edited to reflect that in OP before you were able to refresh it seems.

Also REEEEEEEE ok